Resources/Prototypes/Guidebook/chemicals.yml

```diff
@@ -0,0 +1,60 @@
+- type: guideEntry
+  id: Chemicals
+  name: guide-entry-chemicals
+  text: "/ServerInfo/Guidebook/Chemicals.xml"
+  children:
+  - Elements
+  - Medicine
+  - Narcotics
+  - Pyrotechnic
+  - Toxins
+  - Foods
+  - Botanical
+  - Biological
+  - Others
+  filterEnabled: True
+
+- type: guideEntry
+  id: Elements
+  name: guide-entry-elements
+  text: "/ServerInfo/Guidebook/Chemical Tabs/Elements.xml"
+
+- type: guideEntry
+  id: Medicinal
+  name: guide-entry-medicine
+  text: "/ServerInfo/Guidebook/Medical/Medicine.xml"
+
+- type: guideEntry
+  id: Narcotics
+  name: guide-entry-narcotics
+  text: "/ServerInfo/Guidebook/Chemical Tabs/Narcotics.xml"
+
+- type: guideEntry
+  id: Pyrotechnic
+  name: guide-entry-pyrotechnics
+  text: "/ServerInfo/Guidebook/Chemical Tabs/Pyrotechnic.xml"
+
+- type: guideEntry
+  id: Toxins
+  name: guide-entry-toxins
+  text: "/ServerInfo/Guidebook/Chemical Tabs/Toxins.xml"
+
+- type: guideEntry
+  id: Foods
+  name: guide-entry-foods
+  text: "/ServerInfo/Guidebook/Chemical Tabs/Foods.xml"
+
+- type: guideEntry
+  id: Botanical
+  name: guide-entry-botanical
+  text: "/ServerInfo/Guidebook/Medical/Botanicals.xml"
```

Comment:
This is a duplicate of the existing `Botanicals` page prototype. Consider removing this entry and its respective localization, and instead using the existing prototype.